ProductConstructionService.Api.Tests/ProductConstructionService.Api.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This should go to `tests/`

Reply:
my bad, didn't notice it got put there, moved